src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/NocNotificationHelperTest.java

```diff
@@ -99,8 +100,8 @@ void testBuildPreviousRespondentSolicitorPersonalisation() {
     @Test
     void testBuildNewRespondentSolicitorPersonalisation() {
         Map<String, String> claimantPersonalisation =
-            NocNotificationHelper.buildPersonalisationWithPartyName(caseData, "test_party");
-        assertThat(claimantPersonalisation.size(), is(4));
+            NocNotificationHelper.buildPersonalisationWithPartyName(caseDetails, "test_party");
+        assertThat(claimantPersonalisation.size(), is(5));

```

Comment:
Should this test also test each value in the personalization is correct?

Reply:
Should do, but this isn't my code. I'm just fixing a bug